client/src/main/java/io/pravega/client/segment/impl/SegmentMetadataClient.java

```diff
@@ -49,10 +49,9 @@
      * Deletes all data before the offset of the provided segment.
      * This data will no longer be readable. Existing offsets are not affected by this operations. 
      * The new startingOffset will be reflected in {@link SegmentMetadataClient#getSegmentInfo(String).startingOffset}.
-     * @param segment The segment to truncate.
      * @param offset The offset the segment should be truncated at.
      */
-    abstract void truncateSegment(Segment segment, long offset);
+    abstract void truncateSegment(long offset);
```

Comment:
To confirm, this is an internal API, so we are not breaking API compatibility with this change, are we?

Reply:
Yes, this is an internal API